src/services/AbstractDFG.jl

```diff
@@ -979,13 +979,12 @@ function buildSubgraph(::Type{G},
     return destDFG
 end
 
-# Perhaps IIF.InMemDFGType should perhaps be incorporated as a DFG value, but this has been much debated in the past and hence just left as breadcrum.
-buildSubgraph(dfg::AbstractDFG,
-              variableFactorLabels::Vector{Symbol},
-              distance::Int=1;
-              solvable::Int=0,
-              sessionId::String = "",
-              kwargs...) = buildSubgraph(typeof(LightDFG(params=getSolverParams(dfg))), dfg, variableFactorLabels, distance, solvable=solvable, sessionId=sessionId, kwargs...)
+function buildSubgraph(dfg::AbstractDFG,
+                       variableFactorLabels::Vector{Symbol},
+                       distance::Int=0;
```

Comment:
distance must default to 0 to be compatible with the API

Reply:
Hi @Affie, `distance=1` is most compatible with history of `buildSubgraphFromLabels` that would (from only variable labels) automatically include all factors of inclusive variables.  Need to fix IIF to include the change, thanks.

Reply:
oh you already did thanks JuliaRobotics/IncrementalInference.jl#704